src/components/editors/ExportEditor.js

```diff
@@ -41,6 +38,20 @@ const ExportEditor = () => {
 
   return (
     <Box p={2}>
+
+      <TextField
+        id="standard-basic"
+        label="Theme Name"
+        placeholder={themeName}
+        onChange={( { target: { value } } ) => {
+          // Empty string then set defualt (Overlay 2020-05-21 HH_MM_SS)
+          // Else User given string
+          if ( value.trim() === '' ) {
+            setThemeName( `Overlay ${timestamp()}` )
+          } else setThemeName( value )
```

Comment:
You should either brace both, or one-line both:

```suggestion
          if ( value.trim() === '' ) setThemeName( `Overlay ${timestamp()}` )
          else setThemeName( value )
```